docs/en/operations/settings/settings.md

```diff
@@ -1872,6 +1872,53 @@ Possible values:
 
 Default value: `0`.
 
+## insert_shard_id {#insert_shard_id}
+
+Enables or disables synchronous data insertion into specific shards of [Distributed](../../engines/table-engines/special/distributed.md#distributed) tables.
```

Comment:
`Enables or disables` is appropriate only for settings with two values (enabled/disabled). Maybe we can write something like: `If not zero, specifies the shard of distributed table into which the data will be inserted synchronously.`